### DISCUSSION TITLE
Dudas sobre COORDINAR-PEDIDOS

### DISCUSSION THREAD

**Discussion by Fernanda-Bley** [IIC2133-PUC/2024-1#39] [Tarea 0] [answered]
Me dí cuenta de algo extraño en mi output. 
Cuando imprimía los valores no tenían sentido. 
Digamos que estos son los valores iniciales:
Nave 1: 3 PEDIDOS
Nave 2: 6 PEDIDOS
Nave 3: 12 PEDIDOS
En la primera iteración lo que me pide es que pase un paquete de la nave 3 a la 2 porque el número de paquetes en la nave 2 tiene menos paquetes. 
**Se encuentra en las lineas 28 a 30 del output 3-2**
PAQUETES TRANSFERIDOS: 1
    NAVE 2: 7 PEDIDOS
    NAVE 3: 11 PEDIDOS
PEDIDOS 2 PENDIENTES: 7
Pero en la siguiente línea se contradice, pidiéndome que le pase paquetes a la nave 3 de la nave 1 porque la nave 1 es la que menos tiene. 
**Se encuentra en las lineas 32 a 34 del output 3-2**
PAQUETES TRANSFERIDOS: 1
    NAVE 1: 2 PEDIDOS
    NAVE 3: 12 PEDIDOS
Entonces estoy confundida debo pasarle a quien más paquetes tenga como en la iteración 2 o a quien menos tenga como en la interación 1?
Mi output solo puede obedecer a una de esas dos instrucciones, porque se contradicen, entonces queda:
PAQUETES TRANSFERIDOS: 1
    NAVE 2: 7 PEDIDOS
    NAVE 3: 11 PEDIDOS
De nave 1 a nave 2
PAQUETES TRANSFERIDOS: 1
    NAVE 1: 4 PEDIDOS
    NAVE 3: 10 PEDIDOS
ó
PAQUETES TRANSFERIDOS: 1
    NAVE 2: 5 PEDIDOS
    NAVE 3: 13 PEDIDOS
De nave 1 a nave 2
PAQUETES TRANSFERIDOS: 1
    NAVE 1: 2 PEDIDOS
    NAVE 3: 14 PEDIDOS
¿Con cuál me quedo?

**Comment by PabloNavarroY** [comment, marked as answer]
Los paquetes deben pasarse a quien tenga más paquetes que vayan hacia el planeta indicado.

Por ejemplo, en esta imagen se marcan con rojo los paquetes que van al planeta especificado. Los paquetes se pasan a la nave 1 porque tiene más paquetes rojos que la nave 2.

![image](https://github.com/IIC2133-PUC/2024-1/assets/155672565/d3901930-4287-4508-bca2-9d6535b649d7)
